dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/GetOrderBookSpec.scala

```diff
@@ -0,0 +1,82 @@
+package com.wavesplatform.it.matcher.api.http
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.domain.asset.AssetPair
+import com.wavesplatform.dex.domain.order.OrderType.{BUY, SELL}
+import com.wavesplatform.dex.it.api.RawHttpChecks
+import com.wavesplatform.it.MatcherSuiteBase
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class GetOrderBookSpec extends MatcherSuiteBase with TableDrivenPropertyChecks with RawHttpChecks {
+
+  private val negativeAssets = Table(
+    ("Amount", "Price", "Http status", "Error code", "Message"),
+    ("incorrect", "WAVES", 404, 11534345, "The asset incorrect not found"),
+    ("WAVES", "incorrect", 404, 9440771, "The WAVES-incorrect asset pair should be reversed")
+  )
+
+  private val negativeDepth = Table(
+    ("Value", "Title"),
+    ("2147483648", "More than Int.Max"),
+    ("100.0", "Double"),
+    ("incorrect", "Alphabetical")
+  )
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(
+    s"""waves.dex {
+       |  price-assets = [ "$BtcId", "$UsdId", "WAVES" ]
+       |}""".stripMargin
+  )
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueBtcTx)
+    broadcastAndAwait(IssueUsdTx)
```

Comment:
If you write:
```scala
broadcastAndAwait(IssueBtcTx, IssueUsdTx)
```
the test will take multiple seconds less :)

Reply:
fixed